packages/nimble-components/src/table/tests/table-selection.spec.ts

```diff
@@ -1400,6 +1400,72 @@ describe('Table row selection', () => {
                             jasmine.arrayWithExactContents(expectedSelection)
                         );
                     });
+
+                    it('SHIFT + click only selects the clicked row when the previous click deselected the row clicking the selection checkbox', async () => {
```

Comment:
Had to read this several times to parse it. I think this is clearer.
```suggestion
                    it('SHIFT + click selects only the clicked row if the previous click deselected a row (by clicking the checkbox)', async () => {
```